README.md

```diff
@@ -1,63 +1,160 @@
 
-Some examples need to first be deployed on the network you wish to run them on. To do this simply run `node scripts/deploy examples/${name} ${local|testnet}`. Replace `${var}` appropriately. Cannot be run for the `sent-token` and `deposit-address` examples as no deployment is needed.
+3. Build contracts and tests (The update and install takes a few minutes.):
+   ```
+   npm update && npm install
+   npm run build
+   ```
 
-## Testing
+3. To connect to the axelar network, run `npm install axelarnetwork/axelar-local-dev`.
 
-All examples can be tested with `node scripts/test/{example-name} ${local|testnet} ...params`. See below for what values params can take. All params are optional and have default values and can be ommited.
+4. To run a local node, open `axelar-local-gmp-examples` in a separate terminal and run `node scripts/createLocal`. You’ll need to have this node running to deploy the dapps.
 
-### Sent token
+   The five supported networks are deployed: Moonbeam (ChainID: 2500), Avalanche (chainID: 2501), Fantom (chainID: 2502), Ethereum (chainID: 2503), and Polygon (chainID: 2504) on port 8500.
 
-Run `node scripts/test examples/send-token ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+5. To make sure that the address we use for examples is funded on all five supported testnets, run `node scripts/checkBalances`. We use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` to deploy and test all examples. Alternatively, you can use `--address [<address>]` to specify a different address.
 
-### Deposit Address
+## Deploy and test each example
 
-Run `node scripts/test examples/deposti-address ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+For each example, you can allow the defaults to run or enter:
+
+- `local` or `testnet` for both the test and the deploy steps, except for deposit-address and send-token, which run on testnet.
+
+- `source-chain` and `destination-chain`
+
+- variables such as `message`, `amount`, and `account`
+
+and run the local deploy and test.
 
 ### Call contract
 
-Run `node scripts/test examples/call-contract ${local|testnet} ${source-chain} ${destination-chain} ${message}` to relay a message from `source-chain` to `destination-chain`.
+This dapp relays a message from source-chain to destination-chain.
+
+1. To deploy the dapp, run::
+
+`node scripts/deploy examples/call-contract [<local|testnet>]`
+
+2. To test it, run:
+
+`node scripts/test examples/call-contract [<local|testnet>] [<source-chain] [<destination-chain>] [<message]`
+
+3. To share your code cross-chain, run `yarn call-contract`.
```

Comment:
@ggutoski Is this the correct purpose of `yarn': "to share your code cross-chain"? If not, what is the purpose of this command? 
Since they're all cross-chain, why don't the others need it?

Reply:
I think this is a mistake in the original doc.  Just delete this step.  Let's assume that all `yarn` commands should be removed.